location_field/widgets.py

```diff
@@ -47,7 +47,11 @@ def render(self, name, value, attrs=None):
         attrs = attrs or {}
         attrs['data-location-field-options'] = json.dumps(self.options)
 
-        text_input = super(LocationWidget, self).render(name, value, attrs)
+        # Django added renderer parameter in 1.11, made it mandatory in 2.1
+        try:
+            text_input = super(LocationWidget, self).render(name, value, attrs=attrs, renderer=renderer)
+        except TypeError:
+            text_input = super(LocationWidget, self).render(name, value, attrs=attrs)
```

Comment:
Can we just pass `renderer` whenever it is not `None`? For example:

```python
super_kwargs = {'attrs': attrs}
if renderer is not None:
    super_kwargs['renderer'] = renderer
text_input = super(LocationWidget, self).render(name, value, **super_kwargs)
```

Reply:
Pushed.